scripts/build/arch/arm.sh

```diff
@@ -39,3 +39,22 @@ CT_DoArchTupleValues() {
         CT_TARGET_SYS="${CT_TARGET_SYS}hf"
     fi
 }
+
+CT_DoArchUClibcConfig() {
+    local cfg="${1}"
+
+    CT_DoArchUClibcSelectArch "${cfg}" "arm"
+
+    # FIXME: CONFIG_ARM_OABI does not exist in neither uClibc/uClibc-ng
+    # FIXME: CONFIG_ARM_EABI does not seem to affect anything in either of them, too
+    # (both check the compiler's built-in define, __ARM_EABI__ instead) except for
+    # a check for match between toolchain configuration and uClibc-ng in
+    # uClibc_arch_features.h
+    if [ "${CT_ARCH_ARM_EABI}" = "y" ]; then
+        CT_KconfigDisableOption "CONFIG_ARM_OABI" "${cfg}"
+        CT_KconfigEnableOption "CONFIG_ARM_EABI" "${cfg}"
+    else
+        CT_KconfigDisableOption "CONFIG_ARM_EABI" "${cfg}"
+        CT_KconfigEnableOption "CONFIG_ARM_OABI" "${cfg}"
+    fi
+}
```

Comment:
We only support back to 0.9.33.2, and it also doesn't have `CONFIG_ARM_OABI`.
Maybe we should remove it?